packages/decoupled/src/config/__tests__/decoupledJson.test.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  hasDecoupledJson,
+  getFromDecoupledJson
+} from '../decoupledJson';
+
+describe('this is hasDecoupledJson function test', () => {
+    test('expect decoupled json is not exist', () => {
+        expect(hasDecoupledJson()).toBe(false);
+    });
+});
+
+describe('this is getFromDecoupledJson function test', () => {
+    test('expect decoupled json return default value', () => {
+        const result = getFromDecoupledJson('appPath', './');
+        expect(result).toBe('./');
```

Comment:
@friendken Can you please be more elaborate? I.e. this is just testing the default-value, there should also be a test about getting actual values.